src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -1,6 +1,7 @@
 export default Object.freeze({
   dashboardShowCovid19Alert: 'dashboardShowCovid19Alert',
   facilityLocatorShowCommunityCares: 'facilityLocatorShowCommunityCares',
+  profileShowProfile2: 'profile_show_profile_2.0',
```

Comment:
Apparently because there is a period in the feature flag name it doesn't get converted from snake_case camelCase like the other feature flags do. ¯\_(ツ)_/¯

Reply:
should we consider changing this to `profile_show_profile_2` or `profile_show_profile_20` ?

Reply:
Definitely. I think that if we change it to `profile_show_profile_2` [here](https://github.com/department-of-veterans-affairs/vets-api/blob/ef8d1e8f1bead1b85ef4ca6ebc49609e3967467d/config/features.yml#L30) then we can change it to `profileShowProfile2` here in the frontend.

Reply:
I'd run into weird behavior of the olive branch Gem we use to transform strings but this was a new one for me.